pytest_localserver/http.py

```diff
@@ -31,7 +31,10 @@ def __del__(self):
         self.stop()
 
     def stop(self):
-        self._server.shutdown()
+        try:
+            self._server.shutdown()
+        except AttributeError:
+            pass
```

Comment:
Here's an idea:

```suggestion
        try:
            server = self._server
        except AttributeError:
            pass
        else:
            server.shutdown()
```

This way it won't mask any `AttributeError` that might be raised from somewhere inside the server's `shutdown()` method itself. (I don't know if that would ever happen, but if it does, I think it should be allowed to propagate without a really good reason to do otherwise.)